tests/unit/test_checkpointing.py

```diff
@@ -1353,7 +1345,6 @@ def _test_load_immediate_save(args, model, tmpdir):
 @pytest.mark.parametrize('zero_stage', [0, 1, 2, 3])
 def test_save_before_accum_grad_is_done(tmpdir, zero_stage):
     config_dict = {
-        "train_batch_size": 4,
```

Comment:
Why was the batch size config removed here?

Reply:
This violated F601, which is having a duplicate `dict` key. The value is redeclared later on, so I chose to remove this one to preserve the current behavior.